src/docs/goose-book/src/getting-started/runtime-options.md

```diff
@@ -45,7 +45,7 @@ Metrics:
   --no-status-codes           Do not track status code metrics
 
 Advanced:
-  --test-plan "TESTPLAN"      Defines a more complex test plan ("10,60;0,30")
+  --test-plan "TESTPLAN"      Defines a more complex test plan ("10,60s;0,30s")
```

Comment:
You'll also need to update config.rs at https://github.com/tag1consulting/goose/blob/main/src/config.rs#L210, and then also at https://github.com/tag1consulting/goose/blob/main/src/config.rs#L78

(Run `cargo run --release --example simple -- -h` to be sure the help is properly updated.)

Reply:
Done.